app/src/main/java/com/imdbmovieapp/data/local/entity/FavoriteMovieEntity.kt

```diff
@@ -0,0 +1,15 @@
+package com.imdbmovieapp.data.local.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "movieTable")
+data class FavoriteMovieEntity(
+    @PrimaryKey
+    val id: Int,
+    val title: String,
+    val year: String,
+    val genre: String,
+    val image: String,
+    val isFavorite: Int,
```

Comment:
why is "isFavorite" type Int? naming sounds like a boolean, but if it should be an Int, then maybe name of the parameter needs correction?